assignment01_redux/js/myScript1.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function(){
+    $('#first-name').text('James');
+    $('#last-name').text(' Welu');
+    $('.mcc').css('color', 'blue');
+    $('.mcc').css('font-style', 'italic');
+    $('#hobby').text('College');
+    $('ul').append('<li>Electronics</li>');
+    $('ul').append('<li>Films</li>');

```

Comment:
This looks much more like I expected! And you've got the pull request process all sorted! Awesome! I look forward to seeing your work this week 😃